python/perf-kernels/gemm.py

```diff
@@ -323,6 +339,23 @@ def main():
     global verbose
     args = parse_args()
     verbose = args.v
+
+    if args.model:
+        batch_size = args.b if args.b is not None else 1
+        config_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), "model_configs.json")
+        if args.model.lower() == "all":
+            # Benchmark all models
+            x_vals = model_benchmarking.get_mnk(batch_size=batch_size, seq_len=args.sq, config_file=config_file)
+        else:
+            # Benchmark a specific model
+            x_vals = model_benchmarking.get_mnk(batch_size=batch_size, config_file=config_file, seq_len=args.sq,
+                                                model_name=args.model)
+        benchmark.benchmarks.x_vals = x_vals
+
+    if args.M and args.N and args.K:
```

Comment:
May be add an assert that both MNK and model cannot be provided together (because model is actually fixing MNK so the user likely made a mistake if they provided both)?